helix-core/src/main/java/org/apache/helix/ConfigAccessor.java

```diff
@@ -67,14 +68,31 @@
     // @formatter:on
   }
 
-  private final HelixZkClient zkClient;
+  private final HelixZkClient _zkClient;
```

Comment:
We are not expecting unrelated changes in a PR, right? I think if this name change is put in another PR, it would make this PR easier to review, because you can see the name change brings in noise(so many lines of change in this file).

Reply:
What makes you think that this change is unrelated?
- This change is about ZkClient in ConfigAccessor.
- We have a new constructor that involves _zkClient. I didn't want to create a second private variable called _zkClient and leave zkClient as is for the sake of keeping the diff to a minimum. 

All in all, I think this diff is totally reasonable. Sure, you want to keep the change to a minimum. I appreciate the spirit.